Python/Ask_How_To_Qus_using_Voice/README.md

```diff
@@ -0,0 +1,67 @@
+# Ask 'How to' Questions using Voice
+
```

Comment:
Follow [this](https://github.com/HarshCasper/Rotten-Scripts/blob/master/TEMPLATE_README.md) to format your `README.md` accordingly.

Reply:
👍